python/setup.cfg

```diff
@@ -48,4 +48,4 @@ skip=
 packages = find:
 install_requires =
     numpy
-python_requires = >=3.7,<3.9
+python_requires = >=3.8,<3.10
```

Comment:
gpuCI tests 3.8, 3.9 now - are we okay with bumping the `python_requires` of this package?

Reply:
Yes, I think 3.8+ is acceptable.

Reply:
Also, for now let's require CUDA v11.5+

Reply:
yes, i think that would be good